weave_query/weave_query/ops_domain/artifact_membership_ops.py

```
@@ -1,6 +1,9 @@
+import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)